docs/pages/installation/eks.md

```diff
@@ -4,14 +4,21 @@ title: EKS
 permalink: installation/platforms/eks
 ---
 
-# Quick Start with Amazon Elastic Kubernetes Service (EKS)
+## Quick Start with Amazon Elastic Kubernetes Service (EKS)
 
 In order to provide Meshery with the necessary access to your managed Kubernetes instance, 
-Meshery will need to be assigned a `ServiceAccount`. An existing `ServiceAccount` can be used or a new one created. Ensure that the `ServiceAccount` you use has the `cluster-admin` role assigned.
+Meshery will need to be assigned a `ServiceAccount`. An existing `ServiceAccount` can be used or a new one created. 
+1. Ensure that the `ServiceAccount` you use has the `cluster-admin` role assigned.
+2. Configure Meshery to run on EKS:
+    - <a href="#step1">Automatic Configuration </a>
+    - <a href="#step2">Manual Configuration </a>
+
 
 _Note: Make sure you are able to access EKS with `kubectl` by following the [EKS Guide.](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html){:target="_blank"}_
 
-## Automatic Configuration (Recommended)
+<a name="step1">
+
+#### <b>Automatic Configuration (Recommended)</b>
```

Comment:
I understand if this is a one-off. In general, we'll want to globally restyle, if the current h3 style is not suited for the task.

If it's just this section - I understand. That's no issue.